build.gradle

```diff
@@ -86,7 +86,10 @@ subprojects {
             setFailOnNoMatchingTests(false)
         }
 
-        ignoreFailures(true)
+        // in TC we don't have the CI env so we can manage it in this way
```

Comment:
that will also affect local builds, but CI on GH actions will at least catch it, so I guess it's not too bad

Reply:
maybe there is a specific envvar for TC? That would be better.

Reply:
Thank you so much! About the local build I was aware of it and I agree with, the only thing is that maybe we can try to leverage instead some TC parameter but I preferred to **play it safe**

Reply:
`TEAMCITY_VERSION` seems like a good bet: https://www.jetbrains.com/help/teamcity/predefined-build-parameters.html

Reply:
oki